app/js/main.js

```diff
@@ -99,11 +145,14 @@ fileDrop.addEventListener(DRAG_OVER_EVENT, function (e) {
 });
 
 clearButton.addEventListener(CLICK_EVENT, () => {
-    removeCanvas();
-
-    while (imagesList.children.length > 0) {
+    while (imagesList.firstChild) {
```

Comment:
Null safety is required. When imagesList does not contain a child, bang.
`imagesList?.firstChild`

Reply:
Fixed in [d86f63e](https://github.com/markuszeller/image-stitcher/pull/14/commits/d86f63e7dc56fad948bb704ba5b360685571cfe1)